Maquetado/index.html

```diff
@@ -6,9 +6,13 @@
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="SitePoint">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <h1 class="title">
+  	<img src="./assets/triangle.svg"/>
```

Comment:
since this is a purely aesthetic detail we should avoid adding it to the HTML, we should keep the html as simple as possible. We can use a `::before` pseudo-element instead of adding it as an image here.